rbc/src/main/java/com/library/rbc/model/enums/SuggestionStatus.java

```diff
@@ -0,0 +1,8 @@
+package com.library.rbc.model.enums;
+
+public enum SuggestionStatus {
+    ORDERED,
+    PENDING,
+    IN_LIBRARY,
+    CANCELED
+}
```

Comment:
A suggestion can also be approved or denied.

Reply:
When a book suggestion is approved, employee sees 'pending', 'in library', or 'ordered'. Would it be a better option to add a flag 'is approved' so that one of those three statuses is not overridden?

Reply:
We just aligned with Milena, users will also be able to see "approved" and "denied" as a suggestion status. It will be added in figma soon.